.github/workflows/build.yml

```diff
@@ -1,7 +1,10 @@
 ---
 name: build
 
-on: [push]
+on: [
+  push,
+  pull_request
+]
```

Comment:
Shouldn't this section look like this?
```
on:
  push:
  pull_request:
  repository_dispatch:
    types: [apb]
```

Reply:
Hey yeah.  Where all that jazz go?

Reply:
Oh I see!!  It hasn't made it into `skeleton-python-library` yet.  Weird!

Reply:
Created a PR to get things flowing this way: https://github.com/cisagov/skeleton-python-library/pull/43